client/utils/currency/index.js

```diff
@@ -158,11 +199,9 @@ function formatExchangeRate( from, to ) {
 
 function composeFallbackCurrency( amount, currencyCode, isZeroDecimal ) {
 	// Fallback for unsupported currencies: currency code and amount
-	return sprintf(
-		isZeroDecimal ? '%s %i' : '%s %.2f',
-		currencyCode.toUpperCase(),
-		amount
-	);
+	const numericFormat = isZeroDecimal ? '%1$i' : '%1$.2f';
```

Comment:
Are we worried at all about currencies that have more than 2 decimal places?

E.g. Bahraini Dinar (amongst [other currencies](https://en.wikipedia.org/wiki/List_of_circulating_currencies)) have [3 decimal places](https://en.wikipedia.org/wiki/Bahraini_dinar), and now that El Salvador [uses Bitcoin as one of their currencies](https://en.wikipedia.org/wiki/El_Salvador), the decimal places there can go up to 8 via the [satoshi](https://en.wikipedia.org/wiki/Bitcoin#Units_and_divisibility).

Additionally there are a couple of currencies that will only ever have 1 decimal point — but showing 2 decimal points there is fine I think.

I'm afraid this probably isn't as simple as just using none or 2 decimal places 😬 

I'm not sure how to solve this. The list of currencies with more than 2 decimal places is small — 7 including BTC — so it's not an insurmountable task to maintain our own list of that. Decimal places in currencies don't change often.

What do you think?

Reply:
I'm not sure, and I'd prefer to keep that out of scope for this PR (which had "formatting" in the title but wasn't intended to address numeric formatting – changed to "display").

I feel like I got the impression at some point that we only needed to worry about 2 decimal places and zero decimal – maybe Stripe doesn't support those currencies?

Reply:
> I'm not sure, and I'd prefer to keep that out of scope for this PR (which had "formatting" in the title but wasn't intended to address numeric formatting – changed to "display").

Makes sense 👍 

> maybe Stripe doesn't support those currencies?

I feel like that shouldn't be an issue? I don't know though, I'm completely guessing here.

I think we can just open an issue to track this and not worry about it here.